util/grpc/sanitizer.go

```diff
@@ -5,13 +5,16 @@ import (
 	"regexp"
 	"strings"
 
-	"golang.org/x/net/context"
+	"context"
+
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/status"
 )
 
+type sanitizerKey string
+
 const (
-	contextKey = "sanitizer"
+	contextKey sanitizerKey = "sanitizer"
```

Comment:
The linter complained about `contextKey` being of type string, so I wrapped it in its own type.